debian/patches/0007-add-bt2390-eetf-and-code-refactor-to-opencl-tonemap.patch

```diff
@@ -1658,21 +1658,21 @@ Index: FFmpeg/libavfilter/vf_tonemap_opencl.c
 +    if (ctx->in_desc->comp[0].depth == 16) {
 +        // Assume 16bit is actually 12bit for now as that is what the hardware decoders producing
 +        // and what videos are actually encoded in
-+        av_bprintf(&header, "#define INPUT_QUANTIZATION_OFFSET %ff\n", QUANTIZATION_OFFSET(12));
-+        av_bprintf(&header, "#define INPUT_Y_SCALE %ff\n", INPUT_Y_SCALE(12));
-+        av_bprintf(&header, "#define INPUT_UV_SCALE %ff\n", INPUT_UV_SCALE(12));
++        av_bprintf(&header, "#define INPUT_QUANTIZATION_OFFSET %.13lff\n", QUANTIZATION_OFFSET(12));
++        av_bprintf(&header, "#define INPUT_Y_SCALE %.13lff\n", INPUT_Y_SCALE(12));
++        av_bprintf(&header, "#define INPUT_UV_SCALE %.13lff\n", INPUT_UV_SCALE(12));
 +    } else {
-+        av_bprintf(&header, "#define INPUT_QUANTIZATION_OFFSET %ff\n", QUANTIZATION_OFFSET(ctx->in_desc->comp[0].depth));
-+        av_bprintf(&header, "#define INPUT_Y_SCALE %ff\n", INPUT_Y_SCALE(ctx->in_desc->comp[0].depth));
-+        av_bprintf(&header, "#define INPUT_UV_SCALE %ff\n", INPUT_UV_SCALE(ctx->in_desc->comp[0].depth));
++        av_bprintf(&header, "#define INPUT_QUANTIZATION_OFFSET %.13lff\n", QUANTIZATION_OFFSET(ctx->in_desc->comp[0].depth));
++        av_bprintf(&header, "#define INPUT_Y_SCALE %.13lff\n", INPUT_Y_SCALE(ctx->in_desc->comp[0].depth));
++        av_bprintf(&header, "#define INPUT_UV_SCALE %.13lff\n", INPUT_UV_SCALE(ctx->in_desc->comp[0].depth));
```

Comment:
I thought these extra digits would only improve precision in fp64?

Reply:
The INPUT_QUANTIZATION_OFFSET has a lot of 0 after decimal point and %f will round off too many digits as it will always cut of at 6 digits after the decimal point, but single precision rounds off with significant digits, which means we need a bit more digits to even allow single precision.

Reply:
That makes sense.